### DISCUSSION TITLE
Problem with benefits

### DISCUSSION THREAD

**Discussion by ssavav** [community/community#146298] [GitHub Education] [answered]
### Hi everyone,

Hey! My request was approved like an hour ago, but still I don't have access to some benefits such as a github copilot. Should I wait some time or new problem occurred? Screenshot of my approvement is below. Thanks!
![image](https://github.com/user-attachments/assets/3b775999-414e-487f-987f-9f0dd310cc17)

**Comment by kazeokamali** [comment, marked as answer]
maybe u can find 'Github Copilot'page , and there might be a button named 'start free'. Click and activate ...

**Reply by ssavav** [reply]
yeah, there was a button named "Start free trial", but then it said like copliot is free for u, thanks!
